### DISCUSSION TITLE
"Taproot Assets Trustless Swap"    I want to learn, help

### DISCUSSION THREAD

**Discussion by penglipeng666** [lightninglabs/taproot-assets#932] [Q&A]
https://docs.lightning.engineering/the-lightning-network/taproot-assets/trustless-swap
I want to learn this, but I can't find the information. help me

**Comment by guggero** [comment]
Check out the integration test that does that exact swap: https://github.com/lightninglabs/taproot-assets/blob/main/itest/psbt_test.go#L1656

**Reply by penglipeng666** [reply]
thank You ！
